org.adempiere.base/src/org/compiere/model/MOrg.java

```diff
@@ -48,18 +48,38 @@
 	 */
 	public static MOrg[] getOfClient (PO po)
 	{
-		List<MOrg> list = new Query(po.getCtx(), Table_Name, "AD_Client_ID=?", null)
+		return getOfClient(po.getCtx(), po.getAD_Client_ID());
+	}	//	getOfClient
+
+	/**
+	 * 	Get Active Organizations Of current Client
+	 *	@return array of orgs
+	 */
+	public static MOrg[] getOfClient ()
+	{
+		return getOfClient(Env.getCtx(), Env.getAD_Client_ID(Env.getCtx()));		
+	}
+
+	/**
+	 * 	Get Active Organizations Of Client
+	 *	@param ctx context
+	 *	@param int clientID
+	 *	@return array of orgs
+	 */
+	public static MOrg[] getOfClient (Properties ctx, int clientID)
+	{

```

Comment:
this can be simplified by removing the ctx parameter. use env.getctx() here and change the 2 call above to return getOfClient(po.getAD_Client_ID()) and return getOfClient(Env.getAD_Client_ID(Env.getCtx())